torchao/dtypes/affine_quantized_tensor.py

```diff
@@ -217,6 +237,8 @@ def from_hp_to_intx(
             data = data.to(target_dtype)
         else:
             scale, zero_point = choose_qparams_affine(input_float, mapping_type, block_size, target_dtype, quant_min, quant_max, eps, scale_dtype, zero_point_dtype, preserve_zero, zero_point_domain)
+            if zero_point_domain is None:
+                zero_point = None
```

Comment:
this feels a bit confusing, should choose_qparams_affine return None for zero_point directly?

Reply:
yeah thats better, I got tunnel vision

Reply:
Actually can confirm I cant return an optional tensor from there